api/src/main/java/org/openmrs/module/ModuleUtil.java

```diff
@@ -841,13 +842,21 @@ public static AbstractRefreshableApplicationContext refreshApplicationContext(Ab
 		try {
 			ctx.refresh();
 		}
+		catch (Exception e){
+			//Unload the module that cause the Exception and refresh again to restore
```

Comment:
I thought the ticket requires fixing why the exception is thrown and not handling it